### DISCUSSION TITLE
No zeek/suri logs in elastic

### DISCUSSION THREAD

**Discussion by palichx** [Security-Onion-Solutions/securityonion#10049] [2.4]
Thanks for you work! 
I try to install 2.4 version and there is no logs in elastic. 
Current installation ManagerSearch + Sensor.
looks like problen in package policy. errors in sosetup.log:
```
Setting up Suricata package policy...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
^M  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0^M100   629  100    99  100   530    409   2190 --:--:-- --:--:-- --:--:--  2599
{"statusCode":404,"error":"Not Found","message":"[log] package not installed or found in registry"}

```
the same error in kibana:
![image](https://user-images.githubusercontent.com/80513304/228587703-e545699c-3ec8-4e5f-bea3-8bd5ec8e162c.png)

**Comment by TOoSmOotH** [comment]
Does a so-kibana-restart fix it?

**Reply by palichx** [reply]
No.
<details>

so-kibana-restart
=========================================================================
Restarting kibana...

This could take a while if another Salt job is running.
Run this command with --force to stop all Salt jobs before proceeding.
=========================================================================
so-kibana
so-kibana
local:
----------
          ID: kibanasearchgroup
    Function: group.present
        Name: kibana
      Result: True
     Comment: Group kibana is present and up to date
     Started: 15:32:26.928702
    Duration: 6.87 ms
     Changes:
----------
          ID: kibana
    Function: user.present
      Result: True
     Comment: User kibana is present and up to date
     Started: 15:32:26.936511
    Duration: 26.527 ms
     Changes:
----------
          ID: kibanaconfdir
    Function: file.directory
        Name: /opt/so/conf/kibana/etc
      Result: True
     Comment: The directory /opt/so/conf/kibana/etc is in the correct state
     Started: 15:32:26.965144
    Duration: 1.635 ms
     Changes:
----------
          ID: kibanaconfig
    Function: file.managed
        Name: /opt/so/conf/kibana/etc/kibana.yml
      Result: True
     Comment: File /opt/so/conf/kibana/etc/kibana.yml is in the correct state
     Started: 15:32:26.967000
    Duration: 42.506 ms
     Changes:
----------
          ID: kibanalogdir
    Function: file.directory
        Name: /opt/so/log/kibana
      Result: True
     Comment: The directory /opt/so/log/kibana is in the correct state
     Started: 15:32:27.009708
    Duration: 1.593 ms
     Changes:
----------
          ID: kibanacustdashdir
    Function: file.directory
        Name: /opt/so/conf/kibana/customdashboards
      Result: True
     Comment: The directory /opt/so/conf/kibana/customdashboards is in the correct state
     Started: 15:32:27.011485
    Duration: 1.595 ms
     Changes:
----------
          ID: synckibanacustom
    Function: file.recurse
        Name: /opt/so/conf/kibana/customdashboards
      Result: True
     Comment: The directory /opt/so/conf/kibana/customdashboards is in the correct state
     Started: 15:32:27.013265
    Duration: 64.379 ms
     Changes:
----------
          ID: kibanabin
    Function: file.managed
        Name: /usr/sbin/so-kibana-config-load
      Result: True
     Comment: File /usr/sbin/so-kibana-config-load is in the correct state
     Started: 15:32:27.078004
    Duration: 19.745 ms
     Changes:
----------
          ID: so-kibana
    Function: docker_container.running
      Result: True
     Comment: Created container 'so-kibana'
     Started: 15:32:27.178399
    Duration: 1887.641 ms
     Changes:
              ----------
              container:
                  ----------
                  Networks:
                      ----------
                      bridge:
                          ----------
                          IPConfiguration:
                              ----------
                              new:
                                  not connected
                              old:
                                  automatic
                      sobridge:
                          ----------
                          Aliases:
                              ----------
                              new:
                                  - b2745a7cd980
                              old:
                                  None
                          IPAMConfig:
                              ----------
                              new:
                                  ----------
                                  IPv4Address:
                                      172.17.1.27
                              old:
                                  None
              container_id:
                  ----------
                  added:
                      b2745a7cd9807e38b02e7d6d36e1ca1b8a2f2a934ec1aec9aec50f23b3ebe2cc
              state:
                  ----------
                  new:
                      running
                  old:
                      None
----------
          ID: append_so-kibana_so-status.conf
    Function: file.append
        Name: /opt/so/conf/so-status/so-status.conf
      Result: True
     Comment: File /opt/so/conf/so-status/so-status.conf is in correct state
     Started: 15:32:29.066524
    Duration: 7.171 ms
     Changes:
----------
          ID: osquery_hunt_link
    Function: cmd.script
      Result: True
     Comment: Command 'osquery_hunt_link' run
     Started: 15:32:29.075068
    Duration: 12197.309 ms
     Changes:
              ----------
              pid:
                  140060
              retcode:
                  0
              stderr:
              stdout:
                  SO Hunt link not found, adding link and restarting Kibana container...
                  so-kibana

Summary for local
-------------
Succeeded: 11 (changed=2)
Failed:     0
-------------
Total states run:     11
Total run time:   14.257 s
</details>

**Reply by palichx** [reply]
full sosetup.log in the attachment. maybe it will be useful.
[sosetup.log](https://github.com/Security-Onion-Solutions/securityonion/files/11102699/sosetup.log)

**Reply by weslambert** [reply]
What happens if you navigate to SOC -> Elastic Fleet -> then add a custom log integration to a policy?

Does it prompt you to install the log package, or does it still say it does not exist in the registry?

**Reply by palichx** [reply]
1) Elastic Fleet dashbord:
![image](https://user-images.githubusercontent.com/80513304/228595532-186c5a90-97d6-40fa-b264-8725a6e67cd8.png)
2) Hm.. looks like some problem with internet access...
![image](https://user-images.githubusercontent.com/80513304/228598381-503f09d9-2785-4076-bb00-feb481ef4d8b.png)
i will check it. thanks..

**Reply by weslambert** [reply]
Yes, that is where I was going next. I would confirm you can actually access the package registry. Currently, it requires internet access. IIRC, we are planning on hosting a local package registry in the future.

**Comment by palichx** [comment]
yep. 
That was my fault. thanks for help!
